pkg/hub

```diff
@@ -0,0 +1,22 @@
+[mirrors]
+https://github.com/github/hub/archive/v2.5.1.tar.gz
+
+[vars]
+filesize=990972
+sha512=31a9c721dd29e9a7aee28bccb36621802c6d85dbb7277a2f55365c2fcaec4f4a587e6eb7a6fd9231c7c86acdf13ccad9c7846e12ad6f76812943e8d5eb0d9383
+tarball=hub-2.5.1.tar.gz
+tardir=hub-2.5.1
+pkgver=1
+
+[deps]
+go
+git
+
+[build]
+repo="github.com/github/hub"
+
+dest="$butch_install_dir""$butch_prefix"
+srcdest="$S"/build/"$butch_package_name"
+GOROOT="$butch_root_dir"/opt/go GOPATH="$dest" go get "$repo"
```

Comment:
lol, why do we add a tarball if we circumvent the package manager with `go get` ?

Reply:
hmm KEEP/pkg_skel/go should be updated too

Reply:
well, the next two lines in the build make it so butch keeps track of the final binary. `$butch files hub`
`/bin/hub`
but I see what you mean. I'll see what other distros do

Reply:
or do you mean the tarball download should just be skipped?

Reply:
the philosophy of sabotage is that you should be able to download packages in advance and install/build without internet connection. so it's important that we figure out how to install go programs/libs without relying on go get. just like we don't run "pip install xxx" inside python-xxx. but if there's really no way, then yeah, the download of the tarball is superfluous.